config/manager/kustomization.yaml

```diff
@@ -10,3 +10,7 @@ configMapGenerator:
   name: awx-resource-manager-config
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
+images:
```

Comment:
We probably don't want to commit this kustomization change.  Could .gitignore it, as `IMG=yours make deploy` will keep generating it.